benchmark/python/dot.py

```diff
@@ -283,12 +279,15 @@ def bench_dot(lhs_shape, rhs_shape, lhs_stype, rhs_stype,
         if fw == "mxnet":
             lhs_dns = lhs_nd if lhs_stype == 'default' else lhs_nd.tostype('default')
             rhs_dns = rhs_nd if rhs_stype == 'default' else rhs_nd.tostype('default')
+            if trans_lhs:
+                lhs_nd = lhs_nd.T
+                lhs_dns = lhs_dns.T
             # One warm up run, verify correctness
-            out = dot_func_sparse(lhs_nd, rhs_dns, trans_lhs)
-            out_expected = dot_func_dense(lhs_dns, rhs_dns, trans_lhs)
+            out = dot_func_sparse(lhs_nd, rhs_dns)
+            out_expected = dot_func_dense(lhs_dns, rhs_dns)
             assert_almost_equal(out.asnumpy(), out_expected.asnumpy(), rtol=1e-1, atol=1e-1)
-            sparse_cost = measure_cost(num_repeat, dot_func_sparse, lhs_nd, rhs_nd, trans_lhs)
-            dense_cost = measure_cost(num_repeat, dot_func_dense, lhs_dns, rhs_dns, trans_lhs)
+            sparse_cost = measure_cost(num_repeat, dot_func_sparse, lhs_nd, rhs_nd)
```

Comment:
No this is the opposite of what I meant.. We should move the transpose inside the `measure_cost` for both scipy and mxnet..